composer.json

```diff
@@ -18,7 +18,7 @@
         "php": "^7.1",
         "squizlabs/php_codesniffer": "^3.2",
         "dealerdirect/phpcodesniffer-composer-installer": "^0.4.2",
-        "slevomat/coding-standard": "^4.3.0"
+        "slevomat/coding-standard": "dev-master"
```

Comment:
This needs to become a stable dependency before merging

Reply:
+ requires bump once tagged in Slevomat CS. 👍

Reply:
4.4 is out